invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/RunActionForm.js

```diff
@@ -56,9 +56,20 @@ export class RunActionForm extends Component {
     const jsonData = JSON.parse(resource.default_args);
     const { activeIndex } = this.state;
     return (
-      <>
-        <Formik initialValues={formData} onSubmit={onSubmit}>
-          {(props) => (
+      <Formik initialValues={formData} onSubmit={onSubmit}>
+        {(props) => {
+          const actions_errors = props?.errors
+          const args = props?.values?.args;
+          let cleanedArgs = undefined;
```

Comment:
Needed to clean up the values because of the problem in this scenario:
1. input incorrect date into one of the fields
2. remove everything from that input field and fill in another field
3. the first field is not `null` now, but an empty string (because formik) 
4. empty string is not a valid date --> validation failed

Reply:
are you sure there is nothing in formik that could help with that? I think there was something related to touched values.

Reply:
There is a `touched` field, but it just indicates if the field was touched or not, I would still need to implement the logic of cleaning up the value. Maybe I am missing something, but I didn't see anything useful for this in Formik.

Reply:
it would be more proper to clean the values using formik states and methods. Doing it in the render function, you clean it up every time component is re-rendered.
Let's use the formik API: https://formik.org/docs/api/formik one of the functions, onSubmit, validate maybe? to clean only the touched and empty values and only before sending it to the backend

Reply:
Moved it to the `onSubmit`: https://github.com/inveniosoftware/invenio-administration/pull/234/files#diff-73e47f835ff261a0a2ca77309f2f0b2a4a234f5f83d27ec3f0b487f799bcb746R102 👍